### PR TITLE
[DDK][PSDK] Added the rest of the defines for netkvm

### PR DESCRIPTION
**Pull request by BenNottelling**
## Purpose

Fill in the rest of the NDIS definitions (and others that I came across) needed for netkvm, I've done my best to keep the structure right on everything, 
[CORE-15841]( https://jira.reactos.org/browse/CORE-15841)

### CODE REVIEW

File: sdk/include/psdk/ntddndis.h [hunk 1]
```diff
@@ -443,3 +486,14 @@ typedef struct _NDIS_CO_LINK_SPEED {
 #endif
 
 #endif /* _NTDDNDIS_ */
```
Comment (ThFabba) [inline sdk/include/psdk/ntddndis.h]: This is the header guard. You generally want everything to be above it.

File: sdk/include/psdk/ntddndis.h [hunk 2]
```diff
@@ -331,6 +343,37 @@ typedef ULONG NDIS_OID, *PNDIS_OID;
...
+
+/* New NDIS 6 offload OIDs */
+#define OID_TCP_OFFLOAD_CURRENT_CONFIG                     0xFC01020B /* NDIS 5 handled. Query only */
+#define OID_TCP_OFFLOAD_PARAMETERS                         0xFC01020C /* Set only */
+#define OID_TCP_OFFLOAD_HARDWARE_CAPABILITIES              0xFC01020D /* Query only */
+#define OID_TCP_CONNECTION_OFFLOAD_CURRENT_CONFIG          0xFC01020E /* Query only */
+#define OID_TCP_CONNECTION_OFFLOAD_HARDWARE_CAPABILITIES   0xFC01020F /* Query only */
+#define OID_OFFLOAD_ENCAPSULATION                          0x0101010A
+
+
```
Comment (ThFabba) [inline sdk/include/psdk/ntddndis.h]: Unless the rest of the file already does this, I'd stick with a single empty line.

File: sdk/include/psdk/ntddndis.h (hunk 1 above)
Comment (BenNottelling) [reply sdk/include/psdk/ntddndis.h]: Moved it above the `#endif /* _NTDDNDIS_ */`, I believe that's what you wanted?

File: sdk/include/psdk/ntddndis.h [hunk 3]
```diff
@@ -316,7 +316,19 @@ typedef ULONG NDIS_OID, *PNDIS_OID;
 #define OID_802_11_PRIVACY_FILTER               0x0D010119
 #define OID_802_11_BSSID_LIST_SCAN              0x0D01011A
 #define OID_802_11_WEP_STATUS                   0x0D01011B
+/* New name is supposed to better reflect the extended set of encryption status */
+#define OID_802_11 ENCRYPTION_STATUS            OID_802_11_WEP_STATUS
```
Comment (learn-more) [inline sdk/include/psdk/ntddndis.h]: There shouldnt be a space here, but an underscore? (between `11` and `ENCRYPTION`)

Comment (BenNottelling) [reply sdk/include/psdk/ntddndis.h]: Oh, good catch. Committing now

File: sdk/include/psdk/ntddndis.h [hunk 4]
```diff
@@ -442,4 +484,14 @@ typedef struct _NDIS_CO_LINK_SPEED {
 }
 #endif
 
+/* Full duplex driver */
```
Comment (HBelusca) [inline sdk/include/psdk/ntddndis.h]: Preferrably move these before the terminating `#ifdef __cplusplus`, even after the last `#define NDIS_MINIPORT_64BITS_DMA ...` to get all the defines together.

Comment (BenNottelling) [reply sdk/include/psdk/ntddndis.h]: Sorry, realizing how stupid that was now. Going to resolve this and the other comment on placement now

**Pull request merged**